### DISCUSSION TITLE
Datasource connection for Athena and Redshift Databases using ADFS

### DISCUSSION THREAD

**Discussion by tchung555** [great-expectations/great_expectations#5381] [Community Use Cases]
Does Great Expectations support datasource connection using ADFS for Athena and Redshift databases?  If so, are there examples of how to set datasource connection for them?

**Comment by AFineDayFor** [comment]
Howdy @tchung555, I'm moving this over to the Community Use Cases area of Discussions since it'd fit in more in that area. Though we provide information for use with those backends, I believe the community can best help you investigate using it with AD.
